module/epochs/qc_voter.go

```diff
@@ -86,22 +96,19 @@ func (voter *RootQCVoter) Vote(ctx context.Context, epoch protocol.Epoch) error
 		return fmt.Errorf("could not create vote for cluster root qc: %w", err)
 	}
 
+	expRetry, err := retry.NewExponential(retryMilliseconds)
+	if err != nil {
+		voter.log.Fatal().Err(err).Msg("create retry mechanism")
+	}
+
 	attempts := 0
-	for {
+	err = retry.Do(ctx, retry.WithJitterPercent(retryJitter, expRetry), func(ctx context.Context) error {
 		attempts++
-		log := log.With().Int("attempt", attempts).Logger()
-
-		// for all attempts after the first, wait before re-trying
-		if attempts > 1 {
-			wait := voter.getWaitInterval(attempts - 2) // -2 so that we wait the base interval in the first Sleep
-			log.Info().Msgf("waiting for %s before retry", wait.String())
-
-			select {
-			case <-ctx.Done():
-				return fmt.Errorf("context cancelled: %w", ctx.Err())
-			case <-time.After(wait):
-				// proceed and re-submit vote
-			}
+
+		// retry with next fallback client every 2 attempts
+		if attempts%2 == 0 {
+			log.Info().Msgf("retrying on attempt (%x) with fallback access node", attempts)
```

Comment:
```suggestion
			log.Info().Msgf("retrying on attempt (%d) with fallback access node", attempts)
```

`%x` is hex

Reply:
https://github.com/onflow/flow-go/pull/1395/commits/63038258db5767cf410d7b0178d2c13bf2cf1e35